### PR TITLE
Pktmbuf free

### LINKED ISSUE #435: Adopt rte_pktmbuf_free_bulk()
More recent versions of DPDK add the ability to do a bulk freeing of packets. We should adopt it into Gatekeeper when we upgrade DPDK.

### PR DESCRIPTION
**Pull request by mengxiang0811**
This pull request closes #435.

### CODE REVIEW

**Review by AltraMayor — CHANGES_REQUESTED** [review]
**lib/acl: adopt rte_pktmbuf_free_bulk()**

File: gt/main.c [hunk 1]
```diff
@@ -1691,10 +1691,9 @@ gt_proc(void *arg)
 		 * XXX #71 Do something better here!
 		 * For now, free any unsent packets.
 		 */
-		if (unlikely(num_tx_succ < num_tx)) {
-			for (i = num_tx_succ; i < num_tx; i++)
-				rte_pktmbuf_free(tx_bufs[i]);
-		}
+		if (unlikely(num_tx_succ < num_tx))
+			rte_pktmbuf_free_bulk(&tx_bufs[num_tx_succ],
+				num_tx - num_tx_succ);
```
Comment (AltraMayor) [inline gt/main.c:1696]: Style. Given that you had to break the call `rte_pktmbuf_free_bulk()` into two lines, bring back the braces of the `if` statement.

**Review by AltraMayor — CHANGES_REQUESTED** [review]
**gt: adopt rte_pktmbuf_free_bulk()**

**Comment by mengxiang0811** [comment]
This PR is ready for another review.

**Pull request merged**